web-services-app/src/test/java/edu/unc/lib/boxc/web/services/processing/SingleUseKeyServiceTest.java

```diff
@@ -124,6 +124,16 @@ public void testInvalidateWhenKeyIsNotPresent() throws IOException {
         assertEquals(3, records.size());
     }
 
+    @Test
+    public void testInvalidateTimeBased() throws IOException {
```

Comment:
It'd be good to add a test that does an invalid based on time, but doesn't delete anything (or at least leaves one item left) so we can demonstrate it doesn't just delete everything.